### PR TITLE
release checklist: convenience command and small fix

### PR DESCRIPTION
**Pull request by petrosagg**
Adds a commands to automatically find the release commit and fixes the moved path of the load tests

---
This change is [Reviewable](https://reviewable.io/reviews/materializeinc/materialize/9881)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/05-release.md [hunk 1]
```diff
@@ -32,7 +32,15 @@ production readiness.
 
 - [ ] Choose the commit for the release. The commit should *always* be the
   last commit before noon EST on Monday, even if you start the release process
-  late.
+  late. From an updated `main` branch, use this command to find the release commit:
+
+  ```shell
+  # If you run the release on Monday
```
Comment (umanwizard) [inline .github/ISSUE_TEMPLATE/05-release.md:38]: Did you test these on both macOS and Gnu date?

Comment (petrosagg) [reply .github/ISSUE_TEMPLATE/05-release.md:38]: No, just gnu. Do they work on your Mac?

Comment (benesch) [reply .github/ISSUE_TEMPLATE/05-release.md:38]: Classic:

```
$ date -Iminutes -d 'this monday 12:00 EST'
date: illegal option -- I
usage: date [-jnRu] [-d dst] [-r seconds] [-t west] [-v[+|-]val[ymwdHMS]] ... 
            [-f fmt date | [[[mm]dd]HH]MM[[cc]yy][.ss]] [+format]
```

Comment (umanwizard) [reply .github/ISSUE_TEMPLATE/05-release.md:38]: We can just say to use gdate if you are on macOS. People can install it with brew probably.

Comment (umanwizard) [reply .github/ISSUE_TEMPLATE/05-release.md:38]: confirmed, `brew install coreutils` gives you gdate on macOS

**Comment by benesch** [comment]
SGTM

**Review by umanwizard — APPROVED** [review]
Approved assuming you leave the note about `brew install coreutils` and `gdate`

File: .github/ISSUE_TEMPLATE/05-release.md (hunk 1 above)
Comment (petrosagg) [reply .github/ISSUE_TEMPLATE/05-release.md:41]: Thanks, added the note

**Pull request merged**